buildscripts/kokoro/unix.sh

```diff
@@ -66,9 +66,14 @@ if [[ -z "${SKIP_TESTS:-}" ]]; then
   popd
   pushd examples/example-hostname
   ../gradlew build $GRADLE_FLAGS
-  mvn verify --batch-mode
+  mvn clean verify --batch-mode
```

Comment:
I don't think the clean does anything. We could delete it from the others as well and not lose anything.

Reply:
Yeah, you are right. Maven is built to a separate `target/` directory, it will not use Gradle's result. Deleted those cleans.